src/core/ColorPicker/ColorSwatch.tsx

```diff
@@ -40,8 +40,10 @@ export const ColorSwatch = React.forwardRef<HTMLDivElement, ColorSwatchProps>(
 
     const _style = React.useMemo(
       () =>
-        getWindow()?.CSS?.supports?.(`--swatch-color: ${colorString}`)
-          ? { '--swatch-color': colorString, ...style }
+        getWindow()?.CSS?.supports?.(
+          `--iui-color-swatch-background: ${colorString}`,
```

Comment:
The parameter usage doesn't match that of ColorBuilder.tsx line 90.

Reply:
Not sure what you mean by "doesn't match", they are different variables

Reply:
You just set the CSS to '0' in ColorBuilder rather than using the variable `squareTop`. Here you used the variable `colorString` instead of '#000' or something. Why did you do that?

Reply:
Oh, that part is just for `supports`, it doesn't matter what we use. In `ColorBuilder`, I didn't want to list the full value of the inset variable which is `${squareTop.toString()}% auto auto ${squareLeft.toString()}%`, so I just used `0`. In here, I kept it the same as before, although I guess I could have picked a random color.

Maybe we should standardize the way we check for CSS variables support, but ultimately this doesn't affect functionality in any way.